src/main/webapp/dev/js/common/visibilityOptions.es6

```diff
@@ -373,6 +373,9 @@ function updateVisibilityMessageDiv($containingForm) {
         type: 'POST',
         url,
         data: formData,
+        beforeSend() {
+            $visibilityMessageDiv.html("<img height='25' width='25' src='/images/ajax-preload.gif'/>");
```

Comment:
Comparison of the two:

`ajax-preload.gif`
<img width="1110" alt="ajax-preload" src="https://user-images.githubusercontent.com/6811428/28098785-deefc40a-66ea-11e7-9db8-94ea07dfe740.png">
`ajax-loader.gif`
<img width="1110" alt="ajax-loader" src="https://user-images.githubusercontent.com/6811428/28098794-e6c4531c-66ea-11e7-87fb-6799a50f4346.png">